### PR TITLE
[Docs] Added how to use token for Meshery API in postman

### PR DESCRIPTION
**Pull request by Freedisch**
**Notes for Reviewers**

This PR fixes https://discuss.layer5.io/t/how-do-i-use-the-token-while-making-a-request-to-the-meshery-rest-api/2713/2

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit ae6df67f644990e38b9a224d7c0eb90e5b070e37 at: https://64abeba580902c0740574d96--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit fc8b4cd90e7d2e404d4fc452572be26cec265cc2 at: https://64abeed276295107df2dff30--meshery-docs-preview.netlify.app

**Review by thisiskaransgit — COMMENTED** [review]
@Freedisch 
- Let's keep the steps and title platform agnostic, we can still keep Postman as an example
"How to use the token for requests to Meshery Rest API using API Clients"

- Let's use code block for downloaded token replacing screencshot
```javascript
"meshery-provider": "Meshery",
"token": "your token"
```

File: docs/pages/extensibility/api.md [hunk 1]
```diff
@@ -34,7 +48,26 @@ Using Meshery CLI, you can get a copy of your authentication token by executing
...
+Steps to test Meshery API with Postman:
+
+- Download your token from Meshery UI and copy the token value from the downloaded file, this will be used for authentication
+
+![image](https://github.com/Freedisch/meshery/assets/82499435/2fb42a2b-6cd8-4e1b-a6b0-42ee796d6755)
+
+- In Postman, navigate to the Cookies section.
+- Set the appropriate name and value for the cookie
+
+![Screenshot 2023-05-11 at 8 27 49 PM](https://github.com/Freedisch/meshery/assets/82499435/41ee447c-eb3d-4faf-8028-f83643282925)
```
Comment (leecalcote) [inline docs/pages/extensibility/api.md:64]: We'll need to find a repo-local location for this.

**Comment by l5io** [comment]
🚀 Preview for commit b76e43fb451ea45d189840dbc3128b83c8a00fed at: https://64b12fdb3a4271065b50efa2--meshery-docs-preview.netlify.app

**Comment by vishalvivekm** [comment]
@Freedisch Let's discuss this on Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by leecalcote** [comment]
@asubedy review needed...

**Comment by leecalcote** [comment]
merge conflict

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Merge conflict, @Freedisch

**Comment by Freedisch** [comment]
> Merge conflict, @Freedisch

done

**Comment by l5io** [comment]
🚀 Preview for commit 921ba36e63b38e7547053a3e741d125137820a0b at: https://64f3038ed019ec5ac4374494--meshery-docs-preview.netlify.app

**Pull request merged**